pkgs/development/libraries/libgpod/default.nix

```diff
@@ -27,20 +40,30 @@ stdenv.mkDerivation rec {
     "--with-udev-dir=${placeholder "out"}/lib/udev"
   ] ++ lib.optionals monoSupport [ "--with-mono" ];
 
-  dontStrip = true;
-
-  propagatedBuildInputs = [ glib libxml2 sqlite zlib sg3_utils
-    gdk-pixbuf taglib libimobiledevice ];
+  dontStrip = monoSupport;
 
   nativeBuildInputs = [ autoreconfHook intltool pkg-config ]
     ++ (with perlPackages; [ perl XMLParser ])
     ++ lib.optionals monoSupport [ mono gtk-sharp-2_0 ];
 
-  meta = {
-    homepage = "https://gtkpod.sourceforge.net/";
+  buildInputs = [
+    gdk-pixbuf
+    glib
+    libxml2
+    sg3_utils
+    sqlite
+    taglib
+  ];
+
+  propagatedBuildInputs = [
+    libimobiledevice
+  ];
+
+  meta = with lib; {
+    homepage = "https://sourceforge.net/projects/gtkpod/";
     description = "Library used by gtkpod to access the contents of an ipod";
-    license = "LGPL";
-    platforms = lib.platforms.gnu ++ lib.platforms.linux;
+    license = licenses.lgpl2;
```

Comment:
It appears to be `lgpl21Plus`: https://sourceforge.net/p/gtkpod/libgpod/ci/master/tree/src/itdb_sqlite.c